plugins/huawei-cloud/pom.xml

```diff
@@ -19,7 +19,7 @@
     </parent>
 
     <groupId>org.eclipse.xpanse.plugins</groupId>
-    <artifactId>huaweicloud</artifactId>
+    <artifactId>huawei-cloud</artifactId>
```

Comment:
we must keep the old name itself.  same applies for flexibleengine, regiocloud, plusserver.. they are all just one single word